torch/_inductor/ir.py

```diff
@@ -1146,9 +1176,15 @@ def create(  # type: ignore[override]
 
         reduction_numel = V.graph.sizevars.simplify(sympy_product(reduction_ranges))
 
+        # FIXME there's a bunch of dead code in this method, see https://github.com/pytorch/pytorch/issues/109963
+        # Most of the type-ignore comments below are due to this dead code
+
         def const(idx, val):
```

Comment:
I think the callers are correct but the signature is wrong here, `idx` is shadowed by `inner_fn(idx)` for every use.
```suggestion
        def const(val):
```